src/components/imprint/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Typography } from '@mui/material'
+import Link from 'next/link'
+import MUILink from '@mui/material/Link'
+
+const SafeImprint = () => {
+  return (
+    <div>
+      <Typography variant="h1" mb={2}>
+        Imprint & Disclaimer
+      </Typography>
+      <Typography variant="h3" mb={2}>
+        Information in accordance with section 5 of the Telemedia Act (TMG, Germany):
+      </Typography>
+      <Typography mb={2}>
+        Core Contributors GmbH
+        <br />
+        ℅ Full Node
+        <br />
+        Skalitzer Str. 85-86
+        <br />
+        10997 Berlin Germany
+      </Typography>
+      <Typography mb={4}>
+        Managing directors: Richard Meißner, Tobias Schubotz
+        <br />
+        Contact: info@cc0x.dev
```

Comment:
We should probably make this a `mailto` link for consistency with the privacy page.

Reply:
done!